ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -155,6 +155,17 @@ def take_along_axis(arr, indices, axis):
     return ivy.take_along_axis(arr, indices, axis)
 
 
+@with_supported_dtypes(
```

Comment:
Hey! Thanks for the great work! might be some error caused when resolving merge conflicts, but the formatting and decorators are incorrect at the moment for both functional and tests 😅 could you please fix them? Thanks!

Reply:
@juliagsy yes I got the error and  done 👍 Changes Can you Review it again